doc/For developers/Developer guide/Performance/Testing.md

```diff
@@ -0,0 +1,342 @@
+Contrail Windows Performance Testing
+====================================
+
+This document describes assumptions, utilized tools and conclusions drawn from performance tests of Contrail on Windows Server 2016.
+
+## Introduction
+
+TCP throughput was measured in the following scenarios:
+
+1. Raw Windows Server network stack (abbreviated `Raw` in _Results_ section).
+1. Windows Server containers (WinSrv containers) on 2 compute nodes (abbreviated `Containers` in _Results_ section).
+1. WinSrv containers on 1 compute node (abbreviated `Colocated Containers` in _Results_ section).
+1. WinSrv containers on 2 compute nodes, with Contrail Windows (abbreviated `Containers w/ Contrail` in _Results_ section).
+1. WinSrv containers on 2 compute nodes, traffic tuned to eliminate TCP segmentation (abbreviated `Containers (no seg)` in _Results_ section).
+1. WinSrv containers on 2 compute nodes, with Contrail Windows, traffic tuned to eliminate TCP segmentation (abbreviated `Containers w/ Contrail (no seg)` in _Results_ section).
+
+Test results and scenarios are described in the following sections.
+
+## Results
+
+On `sender` nodes:
+
+| Metric              | Raw      | Containers | Colocated Containers | Containers w/ Contrail | Containers (no seg) | Containers w/ Contrail (no seg) |
+|---------------------|----------|------------|----------------------|------------------------|---------------------|---------------------------------|
+| Throughput (Mbit/s) | 7375.159 | 1592.691   | 1956.786             | TBD                    | 235.852             | 98.734                          |
+| Retransmits         | 257      | 335        | 2333                 | TBD                    | 1                   | 1034                            |
```

Comment:
Isn't Retransmits metric meaningless without knowing the time it took for the test??